README.md

```diff
@@ -32,7 +32,10 @@ If you need to reinstall any resources, make sure to delete this file as well.
 
 3. Run `terraform init`
 
-4. Run `terraform apply`
+4. Get the GKE cluster name and location/region from `platform/variables.tf`. 
```

Comment:
At this step the cluster doesn't exist yet, so how do they get the cluster credentials?